lib/index.js

```diff
@@ -113,7 +115,7 @@ var compareNodeNames = function(a, b) {
  * @return {Element}
  */
 function createElement(name, namespaceURI) {
-    return namespaceURI ?
+    return (namespaceURI && namespaceURI !== XHTML) ?
         document.createElementNS(namespaceURI, name) :
```

Comment:
document.createElementNS(namespaceURI || document.documentElement.namespaceURI, name) can be used